Documentation/MigrateToComposer/MigrationSteps.rst

```diff
@@ -211,13 +257,14 @@ Install extension from version control system (e.g. GitHub, Gitlab, ...)
 ------------------------------------------------------------------------
 
 In some cases, you will have to install a TYPO3 extension, which is not
-available on packagist.org or in the TER. Examples could be:
+available on Packagist or in the TER. Examples could be:
 
 *  non-public extension only used by your company.
 *  you forked and modified an existing extension.
+*  commercial plugin / licensed download / Early Access (EAP)
 
 As first step, you have to define the repository in your
-:file:`composer.json`'s "repositories" section. In this example, you find the
+:file:`composer.json`'s `repositories` section. In this example, you find the
```

Comment:
I am not sure that `composer.json's` is sorrectly rendered. Have you tried it? AFAIK a character directly after a backtick means that the backtick is not recognised.

Reply:
Haven't tried that, the code was initially like this so I didn't specifically check. Can do that later (am on mobile now)